<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QWidget" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="issues_list"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="description_line_edit">
       <property name="placeholderText">
        <string>Give the description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="remove_button">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resolve_button">
       <property name="text">
        <string>Resolve</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
